Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,59 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+
+        StartOver:
+            InteractionManager.PerformInteractions();
+
+            Console.WriteLine("Whould you like to start again?");
+            var answer = Console.ReadLine();
+            if (answer.ToLower() == "y") goto StartOver;
+        }
+    }
+
+
+
+    public static class InteractionManager
+    {
+        public static void PerformInteractions()
+        {
+            var person = new Person
+            {
+                Name = Prompter("What is your full name?"),
+                Age = int.Parse(Prompter("What is your age?")),
+                Weight = double.Parse(Prompter("What is your weight in kg?")),
+                Height = double.Parse(Prompter("What is your height in cm?"))
+            };
+
+            Console.WriteLine($"{person.Name} is {person.Age} years old, his weight is {person.Weight} kg and his height is {person.Height} cm.");
+
+            Console.WriteLine($"{person.Name} has a BMI of {person.BMI}");
+        }
+
+        public static string Prompter(string question)
+        {
+            Console.WriteLine(question);
+            return Console.ReadLine();
+        }
+    }
+
+    public class Person
+    {
+        public string Name { get; set; }
+        public int Age { get; set; }
+        public double Weight { get; set; }
+        public double Height { get; set; }
+        public double BMI { get { return CalculateBmi(); } }
```

Comment:
Multiple things:

* You use setters but they do not really make sense in this way:
  * There is no option to change it in any way.
  * The Name is most likely never going to change, so you probably want to create a new instance instead.
* BMI is not supposed to be all uppercase. Conventions say that this should be Bmi.
  * You'd face problems when having such a thing like BMICategory because either break your own rules or have an ugly thing to look at. :D 
  * Although even Microsoft does that I feel like this could easily be a public method instead. Debatable though. :)
  * Not an issue but way cleaner: `public double Bmi => CalculateBmi(); `

Overall I would say it's still nicely done! :)